nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -110,12 +110,13 @@ $(CsWinRTIncludeWinRTInterop)
     <WriteLinesToFile File="$(CsWinRTResponseFile)" Lines="$(CsWinRTParams)" Overwrite="true" WriteOnlyWhenDifferent="true" />
     <Message Text="$(CsWinRTCommand)" Importance="$(CsWinRTMessageImportance)" />
     <Exec Command="$(CsWinRTCommand)" />
+    <!--To support Visual Studio's FastUpToDate builds, we need to cpature the CsWinRT relationship of *.cs: *.winmd.
+    To ensure that modifications to input winmds are reflected in generated *.cs files, WinRT.cs is used as a build
+    marker and updated to the most recent winmd timestamp. UpToDateCheckBuilt then captures the dependency from
+    winmds to cs files.-->
+    <Touch Files="$(CsWinRTGeneratedFilesDir)/WinRT.cs" Time="%(CsWinRTInputs.ModifiedTime)"/>
```

Comment:
So what time gets set?  Does it go through all the inputs and just chooses one of them or sets with each time?  should we be choosing the latest time?

Reply:
yes, it goes thru all inputs and conditionally sets it, so it gets the latest time.  we can't set to latest time, or it will always be newer than build outputs and will always rebuild.

Reply:
I do see that it is going through all the inputs and getting each input's modified time and setting it on WinRT.cs.  But what is the end result?  Does WinRT.cs get the time of the last file processed or some other time?

Reply:
It's using msbuild batching so each Touch is executed independently and conditionally sets the time of WinRT.cs if and only if the new time is later than the current time.  Net effect - WinRT.cs inherits the time of the newest input winmd.